### PR TITLE
Implements documentation

### PR DESCRIPTION
**Pull request by vinimlo**
This will start the documentation process of this awesome way of using NVim!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Vim Like C IDE
+
+## Installation:
+
+ - You will need to install nvim (neovim). That's a powerful extension of Vim, that can help us with some features!
```
Comment (UserXGnu) [inline README.md]: Dude, first of all, thank you very much. But let's start our review.

Nvim isn't an extension of vim, but a fork of it. Nvim is vim with some important parts of the core project completely rewrited. Please, change that part about nvim being a vim's extension! =-D

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    ```
+	In Arch Linux, you should do: 
+	``` yaourt -S neovim ```
+	
+	In systems based on debian you would do something like: 
+	``` apt-get install neovim ```
+
+ - The second step is to clone this repository to your machine. More specifically in ``` ~/.config/nvim/ ```
+
+ - After that you can run the script in that way: ``` sh config.sh ```, or ``` ./config.sh ```
```
Comment (UserXGnu) [inline README.md]: "after that, you might run the script [...]"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ 
+ - So, you can run ``` nvim init.vim ```, and after that you press ``` :PluginInstall ```. That command will install all of that plugins!
```
Comment (UserXGnu) [inline README.md]: "[...] after that you type [...]"

File: README.md (hunk 1 above)
Comment (vinimlo) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (vinimlo) [reply README.md]: done!

File: README.md (hunk 3 above)
Comment (vinimlo) [reply README.md]: done!

**Comment by vinimlo** [comment]
I've fixed @UserXGnu !

**Comment by UserXGnu** [comment]
thx dude

**Pull request merged**